.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+## Describe your changes
+
+...
+
+## Notes for testing your change
+
+...
+
+## Issue ticket number and link
+
+Make sure you use a Github [keyword](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) when adding a ticket number like below.
+
+Resolves #...
```

Comment:
```suggestion
## Describe your changes

<!-- If your changes resolve an open issue, please make sure to note that here using a GitHub keyword (https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) -- e.g. resolves #123 -->

...

## Notes for testing your change

...
```
I think we can just move the issue linking note into the change description, and we can make it a comment so that it doesn't show in the actual PR